### PR TITLE
*: config capacity used ratio.

### PR DESCRIPTION
**Pull request by siddontang**
@qiuyesuifeng @disksing @shenli

### CODE REVIEW

File: cmd/pd-server/main.go [hunk 1]
```diff
@@ -27,17 +27,19 @@ import (
...
+	etcdAddrs       = flag.String("etcd", "127.0.0.1:2379", "Etcd endpoints, separated by comma")
+	rootPath        = flag.String("root", "/pd", "pd root path in etcd")
+	leaderLease     = flag.Int64("lease", 3, "leader lease time (second)")
+	logLevel        = flag.String("L", "debug", "log level: info, debug, warn, error, fatal")
+	pprofAddr       = flag.String("pprof", ":6060", "pprof HTTP listening address")
+	clusterID       = flag.Uint64("cluster-id", 0, "cluster ID")
+	maxPeerCount    = flag.Uint("max-peer-count", 3, "max peer count for the region")
+	metricAddr      = flag.String("metric-addr", "", "StatsD metric address")
+	metricPrefix    = flag.String("metric-prefix", "pd", "metric prefix")
+	minCapUsedRatio = flag.Float64("min-capacity-used-ratio", 0.4, "min capacity used ratio for choosing store in balance")
```
Comment (shenli) [inline cmd/pd-server/main.go]: We have so many configurations now. How about we use a config file to store these configurations in latter PR?

**Comment by shenli** [comment]
LGTM

File: cmd/pd-server/main.go (hunk 1 above)
Comment (siddontang) [inline cmd/pd-server/main.go]: To be honest, I prefer flag and don't like config file...
But config file is ok too if someone supports it later.

**Comment by xiang90** [comment]
@siddontang I run make test a few thousands times on my local machine and could not reproduce the context timeout issue. Is there a way to reliably reproduce the context timeout?

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**